deploy.md

```diff
@@ -1,59 +1,149 @@
-Here are the instructions to deploy Activity Stream:
+# Overview
 
-#  Have the correct credentials
-# Update code and packages
+At the end of each milestone, we will start a new release process that consists of following tasks.
```

Comment:
You mean "At the beginning of each milestone, we will start a new release process for the LAST milestone"?